pkg/cleantemp/cleantemp.go

```diff
@@ -69,25 +69,39 @@ func CleanTempArtifacts(ctx logContext.Context) error {
 	}
 
 	tempDir := os.TempDir()
-	err = filepath.WalkDir(tempDir, func(path string, d fs.DirEntry, err error) error {
+	dir, err := os.Open(tempDir)
+	if err != nil {
+		return fmt.Errorf("error opening temp dir: %w", err)
```

Comment:
should this be set to a log-level for `--debug`

Reply:
I think because it's reading just the temp directory itself that it's fine to have a full error here. If the top level directory can't be read, we likely have a bit of a problem.